packages/react-composites/src/composites/CallWithChatComposite/state/CallWithChatAdapterState.ts

```diff
@@ -115,26 +115,22 @@ export interface CallWithChatAdapterState extends CallWithChatAdapterUiState, Ca
 /**
  * @private
  */
-export function callWithChatAdapterStateFromBackingStates(
-  callAdapter: CallAdapter,
-  chatAdapter: ChatAdapter
-): CallWithChatAdapterState {
+export function callWithChatAdapterStateFromBackingStates(callAdapter: CallAdapter): CallWithChatAdapterState {
   const callAdapterState = callAdapter.getState();
-  const chatAdapterState = chatAdapter.getState();
 
   return {
     call: callAdapterState.call,
-    chat: chatAdapterState.thread,
+    chat: undefined,
```

Comment:
I don't know that we should be clearing the state here with the other adapter missing. is this because its now a promise and we can't resolve these? does this affect the regular ACS group call with GroupId method of joining for example?

Reply:
Yes, at this state chat adapter could not be created. But this fields will be set when chat adapter promise will be executed:
[    chatAdapter.then((chatAdapter) => {
      chatAdapter.onStateChange(this.onChatStateChange);
      /* @conditional-compile-remove(file-sharing) */
      this.chatAdapter = chatAdapter;
      this.context.updateClientStateWithChatState(chatAdapter.getState());
    });](https://vscode.dev/github/Azure/communication-ui-library/blob/ruslanz/meeting-id-passcode-join/packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts#L181-L186)